src/Ocelot.Discovery.Nacos.csproj

```
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net8.0;net9.0</TargetFrameworks>
```

Please note that [Ocelot](https://www.nuget.org/packages/Ocelot) version [23.4.3](https://www.nuget.org/packages/Ocelot/23.4.3#supportedframeworks-body-tab) targets `net6.0`, `net7.0`, and `net8.0`.
Since the "[nacos-sdk-csharp](https://www.nuget.org/packages/nacos-sdk-csharp)" package version [1.3.10](https://www.nuget.org/packages/nacos-sdk-csharp/1.3.10#supportedframeworks-body-tab) targets `net6.0` and `net8.0`, I believe we should target `net6.0` and `net8.0`.
Finally, I would conclude that we cannot target `net9.0`.

**P.S.** Please note, Ocelot will target `net9.0` in version 24.0 which is currently under development
```suggestion
    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
```